### PR TITLE
feat: Adding a new variant for tap-strava built with the SDK

### PR DESCRIPTION
**Pull request by dluftspring**
All Singer definitions are stored in `/_data/taps/` or `/_data/targets`. The minimal requirement for adding a tap or target will match the following format:

```yaml
description: General description of what the company behind the API does
entity_type: Review the JSON Schema for latest options. Currently api, file, database.
entity_url: URL of the developer documentation or website
label: Properly formatted label of the connector
name: The unique name of the connector
singer_name: The Singer specific name of the connector. Typically tap-<name> or target-<name>
type: tap or target
variants: # an array of variant types
- default: true or false
  maintenance_status: "Options: active, beta, development, inactive, unknown"
  meltano_sdk: true or false
  name: Name of the GitHub/GitLab namespace
  pip_url: git+<git_url>.git or pip instal name
  repo: repo URL
  capabilities:
    - catalog
    - discover
  settings:
    # Can be an empty array if unknown: []
    # Describe the list of supported settings, for example:
    - name: project_ids
      kind: array
      label: Project IDs
      description: Array of project IDs.
    - name: username
      label: Username
      description: Credentials used when for connecting to the source.
    - name: password
      kind: password
      label: Password
      description: Credentials used when for connecting to the source.
```

## Checklist

- [x] Add/update the file in the appropriate folder (`/taps` or `/targets`). The name of the file should match the name of the tap. If there is already one, add a descriptor to the name such as `-search`.
- [x] Add/update the PNG logo image in `/assets/logos/<taps or targets>`. The image name must match the YAML file name.
- [x] Tag `@tayloramurphy` or `@pnadolny13` to flag it for review. Or post to the [#hub](https://meltano.slack.com/archives/C01UGBSJNG5) channel on Meltano slack.

## Reviewer Checklist

- [ ] Validate file against JSON Schema. https://www.jsonschema.net/home is an option.
- [ ] Build website locally and validate everything works.

### CODE REVIEW

**Comment by aaronsteers** [comment]
I'm a Strata user and very excited about this. Thanks, @dluftspring !

**Comment by aaronsteers** [comment]
/plugin-test name=tap-strava pip-url=git+https://github.com/dluftspring/tap-strava.git

**Comment by aaronsteers** [comment]
/test-plugin name=tap-strava pip-url=git+https://github.com/dluftspring/tap-strava.git

**Comment by aaronsteers** [comment]
/test-plugin name=tap-strava pip_url=git+https://github.com/dluftspring/tap-strava.git

**Comment by dluftspring** [comment]
> 

This traceback looks like it's coming from the existing variant on Meltano Hub but the git url looks right to me so 🤷 no idea what's happening

**Comment by aaronsteers** [comment]
@dluftspring - Typo on my side above ("pip-url" vs "pip_url" in my slash command.) 😄

**Comment by aaronsteers** [comment]
@dluftspring - Updated checks above look good! ✅ 

Ignoring the first result, the second run gives the expected output and shows the discovered stream names:

https://github.com/meltano/hub/pull/1108#issuecomment-1369144817

File: _data/meltano/extractors/tap-strava/dluftspring.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+name: tap-strava
+namespace: tap_strava
+pip_url: git+https://github.com/dluftspring/tap-strava.git
+repo: https://github.com/dluftspring/tap-strava
+settings:
+- description: Your Strava application unique identifier.
+  kind: password
+  label: Client ID
+  name: client_id
+- description: Your Strave client secret.
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-strava/dluftspring.yml:25]:
```suggestion
- description: Your Strava client secret.
```

**Comment by pnadolny13** [comment]
@dluftspring this is great! I see the current default listed on the hub is stale, should we make yours the default? I would vote yes! If you agree can you just put your variant name in https://github.com/meltano/hub/blob/06a9b0c5f20eff1ac7c6bdc0da60c3d77cbea104/_data/default_variants.yml#L312 instead of `briansloane`?

**Review by pnadolny13 — APPROVED** [review]

**Pull request merged**